Lib9c/Action/ClaimMonsterCollectionReward2.cs

```diff
@@ -0,0 +1,98 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using Bencodex.Types;
+using Libplanet;
+using Libplanet.Action;
+using Nekoyume.Model.Item;
+using Nekoyume.Model.Mail;
+using Nekoyume.Model.State;
+using Nekoyume.TableData;
+using static Lib9c.SerializeKeys;
+
+namespace Nekoyume.Action
+{
+    [Serializable]
+    [ActionType("claim_monster_collection_reward2")]
```

Comment:
This `type_id` seems conflicted with `Lib9c.Action.ClaimMonsterCollectionReward` class' `type_id` and it will cause error. The `Lib9c.Benchmark`'s failure reason may be this. 🤔

I don't know the relation between class name `type_id` perfectfully but it seems to need the change of `Lib9c.Action.ClaimMonsterCollectionReward`'s `type_id`.

Reply:
The `class name` is not changed to make the diff easier to check. And the `class` before the change is renamed to the same name as `type_id` and archived.
e.g.,
previous
```
[ActionType("some_action")]
public class SomeAction { }
```
next
```
[ActionType("some_action2")]
public class SomeAction { }

[ActionType("some_action")]
public class SomeAction1 { }
```

next
```
[ActionType("some_action3")]
public class SomeAction { }

[ActionType("some_action2")]
public class SomeAction2 { }

[ActionType("some_action")]
public class SomeAction1 { }
```